src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.EcDsa.cs

```diff
@@ -25,5 +28,49 @@ internal static partial class Crypto
         // returns the maximum length of a DER encoded ECDSA signature created with this key.
         [DllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_EcDsaSize")]
         internal static extern int EcDsaSize(SafeEcKeyHandle ecKey);
+
+        // todo: move this to common cng and openssl
+        [DllImport(Libraries.CryptoNative, EntryPoint = "CryptoNative_LookupNidByOidValue")]
+        private static extern int CryptoNative_LookupNidByOidValue(string oidValue, ref int nid);
```

Comment:
This looks like a redefinition of https://github.com/dotnet/corefx/blob/master/src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.ASN1.Nid.cs